redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -20,6 +20,8 @@ public function __construct(rex_managed_media $media)
     }
 
     /**
+     * Generate the given file for the given mediatype. This effectively regenerates cached files.
```

Comment:
Hmm, die Methode nutzt doch den Cache, wenn vorhanden, und regeneriert ihn dann nicht. So ist es zumindest gedacht.
Daher die Frage: Denkst du nur, die Methode arbeitet anders, oder hast du das festgestellt?

Reply:
Ich habe den kommentar reverse engineered, von dem was cache warmup erwartet was diese methode macht 

https://github.com/FriendsOfREDAXO/cache_warmup/blob/20a6217d08ea70966baed00cb8145a41c230caab/lib/generator_images.php#L45

Reply:
Ich denke cache-warmup begnügt sich damit, wenn der Cache schon vorhanden ist. Also will gar keine Regenerierung erzwingen, sondern nur sicherstellen, dass der Cache da ist.

Reply:
Hmm das kann sein, ist @schuer aber vermutlich nicht bewusst (es kann zu edge cases führen mit cache warmup)

Reply:
Ohne tiefer drüber nachgedacht zu haben, finde ich das in Cache Warmup eigentlich ok.
Der Media Manager kümmert sich ja eigentlich selbst drum, dass es keine invaliden Cachedateien gibt.
Welche Edge Cases hast du vor Augen?

Reply:
Cache-Warmup möchte den Cache nicht neu generieren, wenn er schon vorhanden ist. Es sollen nur die fehlenden Cachefiles generiert werden.